host/config/chain_spec_list_devnet.json

```diff
@@ -1,17 +1,17 @@
 [
     {
-        "name": "taiko_dev_l1",
-        "chain_id": 32382,
+        "name": "holesky",
```

Comment:
no need to re-conf the holesky, as there is one in _default.json, everything entries should be the same.